ajax/downVoteAjax.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+include('config/config.php');
+include('config/init.php');
```

Comment:
You should probably bundle all of these includes into one single file (like, put config.php in init.php). There's never a time where you won't want to include config, so there's no need to make it a second call.